doc/doxygen/riot.doxyfile

```diff
@@ -1478,7 +1478,7 @@ FORMULA_TRANSPARENT    = YES
 # The default value is: NO.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-USE_MATHJAX            = NO
+USE_MATHJAX            = YES
```

Comment:
```suggestion
USE_MATHJAX            = NO
```

alle uses shown above decrease readabilty by a lot ( most people do not use the rended output of the doxgen